documentation/modules/cruise-control/proc-deploying-cruise-control.adoc

```diff
@@ -104,6 +104,33 @@ kubectl apply -f _kafka.yaml_
 kubectl get deployments -l app.kubernetes.io/name=strimzi
 ----
 
+[discrete]
+== Auto-created topics
+
+The following table shows the three topics that are automatically created when Cruise Control is deployed. These topics are required for Cruise Control to work properly and must not be deleted or changed.
+
+[cols="20,20,60",options="header",stripes="none",separator=¦]
+|===
+
+m¦Auto-created topic
+¦Created by
+¦Function
+
+m¦strimzi.cruisecontrol.metrics
+¦Strimzi Metrics Reporter
+¦Reports broker metrics to Cruise Control.
+
+m¦strimzi.cruisecontrol.partitionmetricsamples
+¦Cruise Control
+.2+¦Obtains metric samples from partitions every two minutes to generate workload snapshots. To change the sampling interval, edit `metric.sampling.interval.ms` in the Cruise Control deployment configuration. 
```

Comment:
what's this `.2+` ? From where this description is coming? I would get info from the CC Wiki Configurations here https://github.com/linkedin/cruise-control/wiki/Configurations. Changing the sampling interval is not so simple because it's even related to the sampling on the metrics reporter side so you would need to change both. I would avoid to mention it.

Reply:
Yeah the description here doesn't really match what this topic is? See the description in my comment below. I would also remove the config as Paolo says.

Reply:
The `.2+` is Asciidoc notation for a table cell that spans two rows. So, the same description is used for both of the "samples" topics. Is that correct here?

I based the description on the "Overview" page in the Cruise Control Wiki: https://github.com/linkedin/cruise-control/wiki/Overview#sampling-tasks

Ah, I didn't realise the complexity involved. I will remove the part about the sampling interval.